### PR TITLE
Feature: Conditions results tutorial

### PR DESCRIPTION
**Pull request by olekssolo**
- Add tutorials for Conditions results
- Add french localization
- Move tutorial_styles from LuresResults to global-constants

### CODE REVIEW

File: global/global-constants.js [hunk 1]
```diff
@@ -55,6 +55,11 @@ const POPUP_HEIGHT_L = 100;
 const SINGLE_LINE = 50;
 const DOUBLE_LINE = 60;
 const WEATHER_MOON_LINE = 95
+const tutorial_styles = {
+  singleLine: [{backgroundColor: primary_color, height: SINGLE_LINE}],
+  doubleLine: [{backgroundColor: primary_color, height: DOUBLE_LINE}],
```
Comment (RaineXael) [inline global/global-constants.js:60]: Even with these tutorial sizing options, the tutorial text still overflows. I feel this approach to tutorial heights isn't the best, as there could be languages with much larger text density than english. Could there be a way of dynamically setting the height?

I changed these values (SINGLE_LINE & DOUBLE_LINE), and they both seem to not have an affect on the tutorial boxes, on my end at least.
Screenshot_20240426_104732_Expo_Go

File: screens/ConditionsResults.js [hunk 2]
```diff
@@ -178,6 +219,23 @@ walleye:  [
           {expandedFish.includes(fishSpecies) && (
             <FlatList
               ListHeaderComponent={<View style={[flex_style.flex, flex_style.center, flex_style.column, margin_styles.vertical_space_md]}>
+                {reactIfView(currentTutorial == 'ConditionsGuide' && index == 0, <Tooltip
```
Comment (RaineXael) [inline screens/ConditionsResults.js:222]: I feel like these tutorial components could be put into one singular React component, with the props telling it which tutorial it is for. Putting repeated components like this into one would help for code repeatability & code reuse.

**Review by RaineXael — COMMENTED** [review]
The tutorial itself was implemented correctly. There are just some comments on Tooltip text readability when the text overflows (with the french language for example), as well as some code reuse for the tutorial components.

File: global/global-constants.js (hunk 1 above)
Comment (olekssolo) [reply global/global-constants.js:60]: I couldn't find anything about text wrap/flexbox type stuff for tooltips. It's hard to find styling information in general about this dependency. For now I just increased the width

File: screens/ConditionsResults.js (hunk 2 above)
Comment (olekssolo) [reply screens/ConditionsResults.js]: I did this in the newest version, very good idea

File: global/global-constants.js (hunk 1 above)
Comment (olekssolo) [reply global/global-constants.js:60]: Additional note: the LuresForm tutorials have not been adjusted yet

File: authentication/auth.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ export default function Authentication(props) {
     if (!authenticated) {
       checkAuthentication()
     } else {
-      navigate(NAV_LURES_RESULTS)
+      navigate(NAV_LURES_FORM)
```
Comment (stefanoScalzo) [inline authentication/auth.js:24]: lets put this back plz

File: global/environment.js [hunk 4]
```diff
@@ -1,8 +1,8 @@
 export const environment = {
   production: false,
-  host: "http://192.168.2.91:3000/",
-  authHost: "http://192.168.2.91:3001/",
-  messagingHost: "http://192.168.2.91:3002/",
+  host: "http://192.168.0.145:3000/",
```
Comment (stefanoScalzo) [inline global/environment.js:3]: lets put this back plz

File: screens/ConditionsForm.js [hunk 5]
```diff
@@ -187,24 +157,9 @@ const ConditionsForm = () => {
...
-                reactIfView(!tooltipDuplicate,
-                  <View style={[{height:Platform.OS === 'android' ? 50 : 0, width: width}]}></View>
-                  )
-            )}
-          </TooltipChildrenContext.Consumer>
-        </Tooltip>
-        )}
+        <TutorialTooltip conditions={currentTutorial == 'selectSpecies'} style={tutorial_styles.doubleLine} 
+          tutorial='selectSpecies' translations='tutSetSpecies' tutRoute={NAV_CONDITIONS_FORM}
+          setCurrentTutorial={setCurrentTutorial}/>
```
Comment (kalo2711) [inline screens/ConditionsForm.js:162]: That's great!

**Review by kalo2711 — COMMENTED** [review]
Great work here! Just check the comments I left before we merge.

File: assets/structures/2d-Submerged_Items.png
Comment (stefanoScalzo) [inline assets/structures/2d-Submerged_Items.png:1]: did u add these images, we already have them in main

Comment (olekssolo) [reply assets/structures/2d-Submerged_Items.png:1]: I think it's because lure-results-tutorial doesn't have them yet

File: add-lure-modal.js [hunk 6]
```diff
@@ -144,52 +106,15 @@ const AddLureModal = (props) => {
...
-          placement="top"
-          onClose={async () => {setCurrentTutorial(await updateTutorialAndGetNext('store', NAV_REQUEST_LURE_FORM))}}
-        >
-          <TooltipChildrenContext.Consumer>
-            {({ tooltipDuplicate }) => (
-              reactIfView(!tooltipDuplicate,
-                <View style={[{height:Platform.OS === 'android' ? 50 : 0, width: width}]}></View>
-              )
-            )}
-          </TooltipChildrenContext.Consumer>
```
Comment (stefanoScalzo) [inline add-lure-modal.js:162]: shouldnt these be replaced with TutorialTooltip

Comment (olekssolo) [reply add-lure-modal.js:162]: That's weird, the conditions-results version of this file doesn't have the changes from lure-results branch. I'll take care of that

**Pull request merged**